### PR TITLE
DM-1600 Loriot Device Registration - Modify documentation to include UI changes

### PR DESCRIPTION
**Pull request by ashhSAG93**

### CODE REVIEW

**Review by miryalarahul — APPROVED** [review]

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 1]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+
+![Register devices](/images/device-protocols/lora-loriot/loriot-selection.png)
+
+{{< c8y-admon-info >}}
+If Loriot is not one of the available options, your tenant is not subscribed to the relevant applications, see information at the top.
+{{< /c8y-admon-info >}}
+
+In the next window, fill in the required information:
+
+- **Title** Title of the device to be registered
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:66]:
```suggestion
- **Title** Title of the device to be registered.
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 2]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+- **Device EUI** This is the unique identifier for the device. It is a 16 character (8 byte) long hexadecimal number. You can find it on the device itself.
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:67]: is this always the case? (the EUI being on the device)

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 3]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+- **Application EUI** This is a global application ID in the IEEE EUI64 address space that uniquely identifies the application provider of the device. It is a 16 character (8 byte) long hexadecimal number.
+- **Application key** This is an AES-128 application key specific for the device that is assigned to the device by the application owner and is responsible to encrypt. The application key is a 32 character (16 byte) long hexadecimal number.
+- **Connection** Lists all configured Loriot connections in the tenant. The following Application name option is populated based on the selected Loriot connection
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:70]:
```suggestion
- **Connection** Lists all configured Loriot connections in the tenant. The **Application name** option (see below) is populated based on the selected Loriot connection.
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 4]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+- **Application name** Select the appropriate application name under which the device needs to be registered in the Loriot provider
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:71]:
```suggestion
- **Application name** Select the appropriate application name under which the device must be registered in the Loriot provider.
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 5]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+- **Device protocol** Select the appropriate device protocol from the dropdown list. For more information on how to create a device protocol refer to [Creating device protocols](#create-loriot-device-protocols).
+
+![Register devices](/images/device-protocols/lora-loriot/loriot-registration.png)
+
+Click **Register** to submit the device registration request and create the device.
+
+You can verify that the device is really connected by checking that events are actually coming in. You can do so by clicking on a device and opening its **Events** tab. All events related to this device are listed here.
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:78]:
```suggestion
You can verify that the device is connected by incoming events. Click on a device and open its **Events** tab. All events related to this device are listed.
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 6]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+
+For more information on viewing and managing your connected devices, also refer to [Device Management](/users-guide/device-management/).
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:80]:
```suggestion
For more information on viewing and managing your connected devices, also refer to [Device Management](/users-guide/device-management/) in the *User guide*.
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 7]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+
+In order to migrate the device from one LNS connection to another, the device needs to be re-registered.
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:82]:
```suggestion
In order to migrate the device from one LNS connection to another, the device must be re-registered:
```

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md [hunk 8]
```diff
@@ -51,25 +51,39 @@ If there are devices associated with the connection, an error message appears, s
...
+Navigate to the **LPWAN** tab of the Device.
+Click on the **Provider connection** dropdown.
+A prompt will appear stating that in order to migrate the device from one LNS connection to another, you need to re-register the device.
+Click on the **Re-Register** button.
+
+The user is directed to the device registration page where he can perform the re-registration following the steps above and selecting the desired LNS connection.
```
Comment (rssag) [inline content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:88]:
```suggestion

1. Navigate to the **LPWAN** tab of the Device.
2. Click the **Provider connection** dropdown.
3. A prompt will appear stating that in order to migrate the device from one LNS connection to another, you must re-register the device. Click the **Re-Register** button.
4. You are directed to the device registration page where you can perform the re-registration following the steps above and selecting the desired LNS connection. 
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md (hunk 2 above)
Comment (ashhSAG93) [reply content/protocol-integration/lora-loriot-bundle/configure-loriot-credentials-cumulocity.md:67]: Yes

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
to which releases does this apply? @ashhSAG93

**Comment by ashhSAG93** [comment]
> to which releases does this apply? @ashhSAG93

10.16

**Pull request merged**